drivers/bluetooth/hci/Kconfig

```diff
@@ -131,3 +131,19 @@ config BT_DRIVER_QUIRK_NO_AUTO_DLE
 
 	  This has to be enabled when the BLE controller connected is Zephyr
 	  open source controller.
+
+
+config BT_HCI_H4_SETUP
+	bool "Enable HCI vendor-specific Setup function"
+	depends on BT_H4
+	default n
```

Comment:
n is the "default default", so you can remove this line

Reply:
updated.